test/libsolidity/ASTJSON/documentation_triple.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
```

Comment:
Sorry, probably should test for comments on contract too. Maybe `@spec` would have contract level invariants defined here?